tests/sentry/api/endpoints/relocations/test_retry.py

```diff
@@ -45,15 +44,15 @@ def get_test_tarball() -> BytesIO:
 @patch("sentry.analytics.record")
 class RetryRelocationTest(APITestCase):
     endpoint = "sentry-api-0-relocations-retry"
+    method = "POST"
 
     def setUp(self):
         super().setUp()
         self.owner = self.create_user(
             email="owner", is_superuser=False, is_staff=True, is_active=True
         )
-        self.superuser = self.create_user(
-            "superuser", is_superuser=True, is_staff=True, is_active=True
-        )
+        self.superuser = self.create_user(is_superuser=True)
```

Comment:
nit: This doesn't seem related to this refactor? Maybe you forgot to split from the subsequent PR?

Applies to other files as well. Not a blocker (this accomplished the goal of making the PRs easier to review), but maybe worth doing for tidiness.